frontend/src/app/core/header/header.component.ts

```diff
@@ -1,4 +1,8 @@
-import { Component, OnInit } from '@angular/core';
+import { 
+  Component,
+  OnInit,
+  ElementRef 
+} from '@angular/core';
 import { TranslateService } from '@ngx-translate/core';
 
 import { NotificationService } from '../../shared/services/notification/notification.service';
```

Comment:
Incorrect import. Use barrels

Reply:
merged before fix.
Will be made in https://github.com/MatvienkoDO/grid-dynamics-internship-project/pull/145